DOCUMENTATION.md

```diff
@@ -47,6 +47,7 @@ If service `a` should start after service `b`, then `a` will be started as soon
 If `b` goes in a `FinishedFailed` state (finished in an unsuccessful manner), `a` might not start at all. 
 * **`start-delay` = `time`**: Start this service with the specified delay. Check how to specify times [here](https://github.com/tailhook/humantime/blob/49f11fdc2a59746085d2457cb46bce204dec746a/src/duration.rs#L338) 
 * **`stdout` = `STDOUT|STDERR|file-path`**: Redirect stdout of this service. STDOUT and STDERR are special strings, pointing to stdout and stderr respectively. Otherwise, a file path is assumed.
+* **`stdout-rotate-size` = `bytes`**: Chunk size of the file specified in `stdout` in bytes. Once the file grows above the specified size it will be closed and a new file will be created with a suffix `.1`. Once the new file also grows above the specified size it will also be closed and a next one will be created with the next suffix `.2`. This allows adding external log rotation script, which can compress the old logs and maybe move them out to a different storage location.
```

Comment:
Can you also add the default value and how it behaves with the default? Also, it might be more convenient if these were kilobytes or even megabytes? I don't think users need _bytes_ granulation level for logs, it depends on the use case but I would probably use 100MB as a default for my horust configs.

Actually, should we set the default to some sane value? I think in the majority of the use cases, they won't want to set this to 0. Maybe we should put 50 or 100 megs as default, and let users pick 0 if they really want to. What do you think? (I'm aware it's a change of behavior, but it feels like a desirable change).

Reply:
This would break the logic of having simple config files like:
```
stdout = "STDOUT"
```
Not specifying the `stdout-rotate-size` would default it to some value and this contradicts the `STDOUT`, which can't be chunked. So this behavior change would break a bunch of unit tests and potentially a bunch of existing user configs.